action.yml

```diff
@@ -31,6 +31,10 @@ inputs:
     required: false
     default: ''
     description: 'Path to a Namespace.Class.StaticMethod to run to perform the build.'
+  manualExit:
+    required: false
+    default: ''
+    description: 'Suppresses `-quit`. Exit your build method using `EditorApplication.Exit(0)` instead.'
```

Comment:
Currently you're passing the variable directly to the bash script and doing complicated logic (I believe this is the first time we use `+set` in this repo) that differs from other `is set` checks in our bash scripts.

We want to move as much away from the bash scripts as possible (in favour of the higher level TypeScript code) and not pass arbitrary values to it. Normalisation and input should either be guaranteed to be correct or throw an error so that the user knows they've misconfigured something.

Reply:
It should default to false. Having it be the presence of a value wasn’t what I wanted, but was a first pass at using what I found; e.g. I didn’t see an example of boolean handling; `Input.getInput` returns a string. Happy to be pointed in the right direction.

Similarly, the bash script responds to the presence or not of the env var, which is clean from the POV of the separation of parsing the input. But I have no love for this kind of scripting and am happy to be pointed in the direction of how this this effect is achieved elsewhere in the codebase.

Reply:
I'm in a bit of a hurry atm, but basically consider workflow inputs as input for the TS logic, and the TS logic to be responsible for passing in something deterministic to the bash script.

Ideally we'd remove the `string | undefined` in favour of `string` like the rest does, and parse that into a `true` or `false` or `1` or `0` value, and pass that into the script.